www/viewgame

```diff
@@ -1456,7 +1464,8 @@ function submitTime() {
         return;
     }
     // Time is valid, as far as we can tell
-    jsonSend("settime?game=<?php echo $id?>&newtime=" + new_time, "timeSaveMsg", swapTimeSections);
+    var time_note = document.getElementById('timeNote').value;
+    jsonSend("settime?game=<?php echo $id?>&newtime=" + new_time + "&note=" + time_note, "timeSaveMsg", swapTimeSections);
```

Comment:
The time note needs to be URL-encoded with `encodeURIComponent(time_note)`.

And you can use a javascript template literal:
```javascript
`settime?game=<?php echo $id?>&newtime=${new_time}&note=${encodeURIComponent(time_note})`
```

Reply:
Changed this